### DISCUSSION TITLE
Cypress

### DISCUSSION THREAD

**Discussion by Palman6** [cypress-io/cypress#17245] [General]
Hello,
 
I am not sure whether this is a bug with cypress or something else.
Till the version Cypress@6.8.0, everything works but thereafter(>6.8.0 - 7.7.0) when I run the yarn start command to start my local server I get this error(see image).
![moment](https://user-images.githubusercontent.com/57383559/124909117-e5d2ed80-e00b-11eb-8fb9-3268f94da06d.png)

@danielmoncada/angular-datetime-picker version is 11.0.1.

**Comment by tit** [comment]
Hi, @Palman6, from https://docs.cypress.io/guides/references/changelog#6-1-0 version, Cypress.moment has been deprecated.
From https://docs.cypress.io/guides/references/changelog#7-0-0 version, Cypress.moment() has been removed.
